week4/assign1/src/pages/Mypage.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { useEffect, useState } from 'react'
+import { Wrapper } from '../styles/common/CommonStyle'
+import styled from "styled-components";
+import { useNavigate, useParams } from 'react-router-dom';
+import axios from 'axios';
+
+const Mypage = () => {
+  const API_URL = import.meta.env.VITE_APP_URL;
+  const nav=useNavigate();
+  const memberId=useParams().userId;
```

Comment:
난 보통 이럴때
```suggestion
  const { userId } = useParams();
```
요로케 가져와서 userId 변수를 활용하는편이얌

Reply:
옹 이렇게 가져올 수도 있구나!!